### ISSUE TITLE
if let - else if bug

### ISSUE THREAD

**Issue by marcellerusu** [marcellerusu/coil-lang#28]
```

def HTMLElement:prototype.to_ast()
  if let kind = this:getAttribute("data-kind")
    let node = {kind: kind}
    for elem of this:children.filter(|elem| elem:hasAttribute("data-attr"))
      node.(elem:getAttribute("data-attr")) = elem.to_ast()
    end
    return node
  else if this.query(":scope > [data-value]") -- this breaks, if I turn this into end / if, it works
    return this:textContent
  else if this:getAttribute("data-array") == "true"
    return this.query_all(":scope > [data-kind]").map(_.to_ast()).into([])
  else
    return this:children.0.to_ast()
  end
end

```

**Comment by marcellerusu** [comment]
yikes.. this might just be a general problem with else-if !!!! :(

**Comment by marcellerusu** [comment]
fixed with 0.1.27

**Comment by marcellerusu** [comment]
issue was that else-if wasn't called Meta.as_bool on the condition
